react/week2/todo-app/src/Todo.js

```diff
@@ -0,0 +1,24 @@
+import React, {useState} from 'react';
+import todoList from "./TodoList";
+
+const Todo = () => {
+  const [todos, setTodos] = useState(todoList)
+  const [check,setCheck] = useState(false)
+  const newTodo = {id: todos.length+1, title:'random text'}
+  const handleClick = () =>{
+    setTodos(todos.concat(newTodo));
+  }
+  const handleDelete = (id) =>{
+    setTodos(todos.filter((todo)=>todo.id!==id));
+  }
+  return (
+    <div>
+       <button onClick={handleClick}>Add Todo</button>
+      {todos.map((todo)=><><li key={todo.id} style={{textDecorationLine: check ? 'line-through' : 'none'}}><input type="checkbox" onClick={() => setCheck(!check)}/>{todo.title}<button onClick={()=>handleDelete(todo.id)}>Delete</button></li></>)}
```

Comment:
Looks great! Only small comment is you don't need to have the React Fragment `<></>` wrapping the `<li></li>` since that is wrapping it all & missing the conditional if there are _"No items"_

Reply:
done